### PR TITLE
Cheat sheet tags for patterns and label expressions

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**
Cheat Sheet PR: https://github.com/neo4j/docs-cheat-sheet/pull/203

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1125.surge.sh

Updated pages:
- [MATCH](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/clauses/match)
- [Fixed length patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/fixed-length-patterns)
- [Non-linear patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/non-linear-patterns)
- [Shortest paths](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/shortest-paths)
- [Variable length patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/variable-length-patterns)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1125.surge.sh

Updated pages:
- [MATCH](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/clauses/match)
- [Fixed length patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/fixed-length-patterns)
- [Non-linear patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/non-linear-patterns)
- [Shortest paths](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/shortest-paths)
- [Variable length patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/variable-length-patterns)

File: modules/ROOT/pages/patterns/shortest-paths.adoc [hunk 1]
```diff
@@ -157,12 +159,14 @@ If there had been only four possible paths between the two Stations, then only t
 To return all paths that are tied for shortest length, use the keywords `ALL SHORTEST`:
 
 .Query
+// tag::patterns_shortest_paths_shortest_k[]
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/patterns/shortest-paths.adoc]:
```suggestion
// tag::patterns_shortest_paths_all_shortest[]
```

File: modules/ROOT/pages/patterns/shortest-paths.adoc [hunk 2]
```diff
@@ -157,12 +159,14 @@ If there had been only four possible paths between the two Stations, then only t
...
 [source,cypher]
 ----
 MATCH p = ALL SHORTEST (wos:Station)-[:LINK]-+(bmv:Station)
 WHERE wos.name = "Worcester Shrub Hill" AND bmv.name = "Bromsgrove"
 RETURN [n in nodes(p) | n.name] AS stops
 ----
+// end::patterns_shortest_paths_shortest_k[]
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/patterns/shortest-paths.adoc]:
```suggestion
// end::patterns_shortest_paths_all_shortest[]
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1125.surge.sh

Updated pages:
- [MATCH](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/clauses/match)
- [Fixed length patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/fixed-length-patterns)
- [Non-linear patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/non-linear-patterns)
- [Shortest paths](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/shortest-paths)
- [Variable length patterns](https://neo4j-docs-cypher-1125.surge.sh/cypher-manual/5/patterns/variable-length-patterns)

**Review by rsill-neo4j — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**